volume/local/local.go

```diff
@@ -144,22 +152,27 @@ type Volume struct {
 	driverName string
 }
 
+// Name returns the volume name
 func (v *Volume) Name() string {
 	return v.name
 }
 
+// DriverName returns the driver that created the volume.
 func (v *Volume) DriverName() string {
 	return v.driverName
 }
 
+// Path returns the data location.
 func (v *Volume) Path() string {
 	return v.path
 }
 
+// Mount returns the location where data is.
 func (v *Volume) Mount() (string, error) {
 	return v.path, nil
 }
 
+// Unmount dissociates this local volume.
```

Comment:
Umount is for satisfying the `Volume` interface and does not do anything in this driver.